state/redis/redis.go

```diff
@@ -99,6 +102,24 @@ func parseRedisMetadata(meta state.Metadata) (metadata, error) {
 		m.maxRetryBackoff = time.Duration(parsedVal)
 	}
 
+	m.failover = defaultFailover
+	if val, ok := meta.Properties[failover]; ok && val != "" {
+		failover, err := strconv.ParseBool(val)
+		if err != nil {
+			return m, fmt.Errorf("redis store error: can't parse failover field: %s", err)
+		}
+		m.failover = failover
```

Comment:
Probably don't need this as the default value for bool is false

Reply:
what do you mean? Remove the `m.failover = failover` and add `m.failover, err := strconv.ParseBool(val)`

Reply:
This line:

m.failover = defaultFailover

Reply:
Ok now, I thought it could remind the user it's false default.